common/src/main/resources/META-INF/resources/import-thunibib.xed

```diff
@@ -62,6 +62,25 @@
     </xed:repeat>
   </xed:template>
 
+  <xed:template id="thunibib.accessRights">
+    <xed:bind xpath="mods:classification[@classID='accessrights']">
+      <div class="form-group form-inline">
+        <label for="{xed:generate-id()}" class="mycore-form-label">
+          <xed:output value="concat(i18n:translate('ubo.accessrights'), ':')" />
+        </label>
+        <select id="{xed:generate-id()}" class="{$xed-validation-marker} mycore-form-input">
+          <option value="">
+            <xed:output i18n="search.select" />
+          </option>
+          <xed:include uri="xslStyle:items2options:classification:editor:-1:children:accessrights" />
+        </select>
+        <a tabindex="0" class="btn btn-secondary info-button" role="button" data-toggle="popover" data-placement="right" data-content="{i18n:ubo.help.accessrights}">
+          <i class="fas fa-info"/>
+        </a>
+      </div>
+    </xed:bind>
+  </xed:template>
+
   <xed:template id="thunibib.typeOfResource">
```

Comment:
das typeOfResource-Template sollten wir auch direkt aus der UBO nutzen können, oder?

Reply:
geht leider nicht, da es dort in import-editor.xed liegt und diese Datei in ThUniBib überschrieben wird